security/libgcrypt-tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import build, distro, git, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class libgcrypt(Test):
+
+    """
+    libgcrypt-testsuite
+    :avocado: tags=security,testsuite
+    """
+
+    def setUp(self):
+        '''
+        Install the basic packages to support libgcrypt
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        deps = ['gcc', 'make']
+        if detected_distro.name in ['rhel', 'SuSE', 'fedora', 'centos',
+                                    'redhat']:
```

Comment:
please remove `redhat`  as distribution  check as after `avocado 50` it is depreciated